server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -118,35 +138,29 @@ public void setupEach() throws IOException {
         .append("role", "viewer")
         .append("avatar", "https://gravatar.com/avatar/08b7610b558a4cbbd20ae99072801f4d?d=identicon");
 
-
     userDocuments.insertMany(testUsers);
     userDocuments.insertOne(sam);
 
     userController = new UserController(db);
   }
 
-  @AfterAll
-  public static void teardown() {
-    db.drop();
-    mongoClient.close();
-  }
-
   @Test
-  public void GetAllUsers() throws IOException {
+  public void getAllUsers() throws IOException {
```

Comment:
Almost every method name in this class started with an upper case letter, when only classes should start with upper case letters. That was kinda tedious to fix, actually.

Reply:
Wahoo! Those names have been weird for a while—thanks for decapitalizing them.